### PR TITLE
fix: nextid other account tip && rarible owner filter

### PR DESCRIPTION
**Pull request by Lanttcat**
- fix: should change tip when account not have wallet bound
- fix: rarible owner filter

## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/web3-providers/src/rarible/index.ts [hunk 1]
```diff
@@ -171,7 +171,9 @@ export class RaribleAPI implements NonFungibleTokenAPI.Provider {
                 hasNextPage: false,
             }
 
-        const data = asset.items.map((asset) => createERC721TokenFromAsset(asset.contract, asset.tokenId, asset))
+        const data = asset.items
+            .map((asset) => createERC721TokenFromAsset(asset.contract, asset.tokenId, asset))
+            .filter((x) => x.info?.owner?.toLowerCase() === from.toLowerCase())
```
Comment (guanbinrui) [inline packages/web3-providers/src/rarible/index.ts:176]: `isSameAddress`

File: packages/mask/src/plugins/NextID/components/NextIdPage.tsx [hunk 2]
```diff
@@ -143,10 +143,17 @@ export function NextIdPage({}: NextIDPageProps) {
...
+                    <Box className={classes.tip}>
+                        <Typography>{t.connect_wallet__other_user_tip_intro()}</Typography>
+                    </Box>
+                )}
+                {isOwn && (
+                    <Box className={classes.tip}>
+                        <Typography>{t.connect_wallet_tip_intro()}</Typography>
+                        <Typography>{t.connect_wallet_tip()}</Typography>
+                    </Box>
+                )}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/NextID/components/NextIdPage.tsx]:
```suggestion
                {isOwn ? A : B}
```

File: packages/web3-providers/src/rarible/index.ts (hunk 1 above)
Comment (Lanttcat) [reply packages/web3-providers/src/rarible/index.ts:176]: Current can‘t import isSameAddress.

**Comment by wenluomask** [comment]
Checked for next.id part.

File: packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx [hunk 3]
```diff
@@ -261,7 +262,11 @@ export function CollectionList({ address }: { address: string }) {
     return (
         <Box>
             {(collections ?? []).map((x, i) => {
-                const renderCollectibles = collectibles.filter((c) => c.contractDetailed.address === x.address)
+                const renderCollectibles = collectibles.filter(
+                    (c) =>
+                        c.contractDetailed.address === x.address ||
```
Comment (guanbinrui) [inline packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx]:
```suggestion
                        isSameAddress(c.contractDetailed.address, x.address) ||
```

File: packages/web3-providers/src/NFTScan/index.ts [hunk 4]
```diff
@@ -101,8 +109,8 @@ export class NFTScanAPI implements NonFungibleTokenAPI.Provider {
         }>('getAllNftByUserAddress', {
             page_size: size,
             page_index: page + 1,
-            use_address: from,
-            erc: 'erc721',
+            user_address: from,
+            erc: pageInfo?.type ?? 'erc721',
```
Comment (guanbinrui) [inline packages/web3-providers/src/NFTScan/index.ts:113]:
```suggestion
            type: pageInfo?.type ?? 'erc721',
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**